plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/plan/PostgreExecutionPlan.java

```diff
@@ -92,8 +92,30 @@ public String getPlanQueryString() {
         if (oldQuery) {
             return "EXPLAIN " + (verbose ? "VERBOSE " : "") + query;
         } else {
-            boolean doAnalyze = CommonUtils.toBoolean(this.configuration.getParameters().get(PostgreQueryPlaner.PARAM_ANALYSE));
-            return "EXPLAIN (FORMAT XML" + (doAnalyze ? ", ANALYSE" : "") + ") " + query;
+            Map<String, Object> parameters = this.configuration.getParameters();
+            StringBuilder explainStat = new StringBuilder(64);
+            explainStat.append("EXPLAIN (FORMAT XML");
```

Comment:
In the original issue user asked:
> I also ask you to give the opportunity to see the full query plan in the form of text, the FORMAT TEXT option

Maybe we should make FORMAT parameter too with options TEXT | XML | JSON | YAML

Reply:
Another issue could be created for this.